src/OrchardCore.Modules/OrchardCore.ContentFields/GraphQL/Fields/ContentFieldsProvider.cs

```diff
@@ -141,22 +150,12 @@ public FieldTypeIndexDescriptor GetFieldIndex(ContentPartFieldDefinition field)
         return new FieldTypeIndexDescriptor
         {
             Index = fieldDescriptor.Index,
-            IndexType = fieldDescriptor.IndexType
+            IndexType = fieldDescriptor.IndexType,
         };
     }
 
-    public bool HasFieldIndex(ContentPartFieldDefinition field) =>
-        _contentFieldTypeMappings.TryGetValue(field.FieldDefinition.Name, out var fieldTypeDescriptor) &&
+    public bool HasFieldIndex(ContentPartFieldDefinition field)
+        => _contentFieldTypeMappings.TryGetValue(field.FieldDefinition.Name, out var fieldTypeDescriptor) &&
         fieldTypeDescriptor.IndexType != null &&
         !string.IsNullOrWhiteSpace(fieldTypeDescriptor.Index);
-
-    private sealed class FieldTypeDescriptor
-    {
-        public string Description { get; set; }
-        public Type FieldType { get; set; }
-        public Type UnderlyingType { get; set; }
-        public Func<ContentElement, object> FieldAccessor { get; set; }
-        public string Index { get; set; }
-        public Type IndexType { get; set; }
-    }
```

Comment:
I would prefer to leave this class here as a nested type, because it is an implementation detail of the `ContentFieldsProvider` only.